### PR TITLE
Chromium Chronicle 22

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- Chromium Chronicle 22

Aiming to publish 6/28/2021

### CODE REVIEW

**Review by atetubou — APPROVED** [review]

**Comment by robdodson** [comment]
@petele who usually reviews these? I've been trying to do a better job of adding assignees to tickets, so I'd like to assign a reviewer if possible.

**Comment by petele** [comment]
@robdodson - they usually come pre-reviewed since they're from the internal Chromium Chronicle... I've typically asked Joe because they're quick and easy, but happy for anyone to review them. This one is now ready to go.

**Comment by petele** [comment]
@rachelandrew or @jpmedley - can one of you take a peek at this and merge? thanks.

File: site/en/blog/chromium-chronicle-22/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Get the code
+
+Run `fetch chromium` in an empty directory. The `fetch` command can also be
+used to get the code for other repositories like v8, or webrtc. See
+`fetch help` for a full list.
+
+## Upload a change for review
+
+Each CL corresponds to a single branch in git, so start by running
```
Comment (rachelandrew) [inline site/en/blog/chromium-chronicle-22/index.md:32]:
```suggestion
Each Change List (CL) corresponds to a single branch in git, so start by running
```

File: site/en/blog/chromium-chronicle-22/index.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+`git new-branch <branch_name>` to create a new branch. Commit as many
+changes as you like to this branch and run `git cl upload` when it's ready
+for review.
+
+`git cl issue` will let you know what CL your branch is associated with.
```
Comment (rachelandrew) [inline site/en/blog/chromium-chronicle-22/index.md:37]:
```suggestion
`git cl issue` will let you know which CL your branch is associated with.
```

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**